### ISSUE TITLE
[Suggestion] Optifine warning

### ISSUE THREAD

**Issue by theishiopian** [TheCyberBrick/The-Betweenlands#1041]
Virtually half the bugs and issues reported on the discord and on github seem to have to do with optifine's fast render feature. A simple fix for this is to, on loading into the betweenlands, check if optifine is loaded, and if so, check the ofoptions.txt file for if fast render is enabled. If it is, print a message to the chat that explains whats happening.

**Comment by PetasiGit** [comment]
People should really just read the README on this site. 

https://github.com/Angry-Pixel/The-Betweenlands/blob/1.12-dev/README.md

![E1EC2EE7-5247-4CDB-9FD1-6D9724E33664](https://user-images.githubusercontent.com/32495545/81479773-6ec97b80-91f3-11ea-952e-577951c86abd.jpeg)

**Comment by theishiopian** [comment]
True, but one thing you can always count on in dev is for users to not read the manual.
